config/locales/en.yml

```diff
@@ -257,6 +257,9 @@ en:
               reused:
                 one: "has been used before. Please choose one that is different from your last one."
                 other: "has been used before. Please choose one that is different from your last %{count}."
+              match: >
+                Confirm new password.
+                Password should match with the one you have typed in New password field.
```

Comment:
I'm not sure but I think it should be ok. See [here](https://crowdin.com/translate/openproject/185/en-de#52478). It only seems to remove line breaks. So if you rely on line breaks in the translation it might be a bad idea.